src/viking.js

```diff
@@ -1,11 +1,55 @@
 // Soldier
-class Soldier {}
+class Soldier {
+
+  constructor(health, strength){
+    this.health=health;
+    this.strength=strength;
+  }
+  
+  attack(){
+    return this.strength;
+  }
+  
+receiveDamage(thedamage){
+    
+      this.health= this.health - theDamage;
+      return "Soldiers never dies";
+      
+        
+      }
+  };
```

Comment:
Make sure you copy/paste variable names! On line 13 you have spelling mistake.

Also always read errors you get/read the task very carefully. Task says:
-receiveDamage() method  shouldn't return anything